`c33200` is not in the tax system

When running a test, I got this error just after installing the new 0.72.3 version:
```
=========================== FAILURES ============================
_________________________ test session __________________________
/Users/.../OpenFisca-US/MA/j21-id3.test.yaml:
Test 'Tax unit with taxsimid 3 from j21.its.csv':
You tried to calculate or to set a value for variable 'c33200', but it
    was not found in the loaded tax and benefit system 
    (openfisca-us@0.72.3).
Are you sure you spelled 'c33200' correctly?
If this code used to work and suddenly does not, this is most 
    probably linked to an update of the tax and benefit system.
Look at its changelog to learn about renames and removals and 
    update your code. If it is an official package, it is probably at
    <https://github.com/openfisca/openfisca-us/blob/master/CHANGELOG.md>.
====================== short test summary info =====================
FAILED j21-id3.test.yaml::
======================== 1 failed in 0.23s =======================
```

Here is where `c33200` appears in the code:
```
(openfisca-us) code% grep -r --include="*py" c33200 openfisca-us-master
openfisca-us-master/openfisca_us/tools/dev/taxcalc/calcfunctions.py:    c33200 = c33000 * 0.01 * crate
openfisca-us-master/openfisca_us/tools/dev/taxcalc/calcfunctions.py:        CDCC_refund = c33200
openfisca-us-master/openfisca_us/tools/dev/taxcalc/calcfunctions.py:        c07180 = min(max(0.0, c05800 - e07300), c33200)
openfisca-us-master/openfisca_us/variables/gov/irs/credits/cdcc/cdcc_refund.py:            return tax_unit("c33200", period)
```

I didn't see any mention of the `c33200` variable being removed in the CHANGELOG.

Has the python file containing this variable's formula gone missing?

--- a/changelog.yaml
+++ b/changelog.yaml
@@ -834,3 +834,7 @@
     fixed:
     - A bug causing itemisation logic to fail.
   date: 2022-06-10 13:24:12
+- bump: patch
+  changes:
+    fixed:
+    - Point `cdcc_refund` to `cdcc` not the deprecated `c33200`.

--- a/openfisca_us/tests/policy/baseline/irs/credits/cdcc/cdcc_refund.yaml
+++ b/openfisca_us/tests/policy/baseline/irs/credits/cdcc/cdcc_refund.yaml
@@ -1,0 +1,13 @@
+- name: CDCC was refundable in 2021.
+  period: 2021
+  input:
+    cdcc: 1
+  output:
+    cdcc_refund: 1
+
+- name: CDCC was not refundable in 2022.
+  period: 2022
+  input:
+    cdcc: 1
+  output:
+    cdcc_refund: 0

--- a/openfisca_us/variables/gov/irs/credits/cdcc/cdcc_refund.py
+++ b/openfisca_us/variables/gov/irs/credits/cdcc/cdcc_refund.py
@@ -13,6 +13,6 @@ class cdcc_refund(Variable):
     def formula(tax_unit, period, parameters):
         cdcc = parameters(period).irs.credits.cdcc
         if cdcc.refundable and not cdcc.abolition:
-            return tax_unit("c33200", period)
+            return tax_unit("cdcc", period)
         else:
             return 0

